.github/workflows/artifact-cleanup.yaml

```
@@ -0,0 +1,32 @@
+        branches:
+            - '**FIT**'
+
+jobs:
+    list-digests:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Set up gcloud
+              uses: google-github-actions/setup-gcloud@v2
+              with:
+                  project_id: ${{ secrets.PROJECT_ID }}
+                  service_account_key: ${{ secrets.GCP_SA_KEY }}
```

actionlint (1.7.4)

input "service_account_key" is not defined in action "google-github-actions/setup-gcloud@v2". available inputs are "install_components", "project_id", "skip_install", "version"

(action)